[2024-08-05T00:20:01.571+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-05T00:20:01.601+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ingest_api_data.check_db scheduled__2024-08-05T00:10:00+00:00 [queued]>
[2024-08-05T00:20:01.608+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ingest_api_data.check_db scheduled__2024-08-05T00:10:00+00:00 [queued]>
[2024-08-05T00:20:01.609+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-08-05T00:20:01.621+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_db> on 2024-08-05 00:10:00+00:00
[2024-08-05T00:20:01.632+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ingest_api_data', 'check_db', 'scheduled__2024-08-05T00:10:00+00:00', '--job-id', '130', '--raw', '--subdir', 'DAGS_FOLDER/ingest_api_data.py', '--cfg-path', '/tmp/tmpaq0167yg']
[2024-08-05T00:20:01.634+0000] {standard_task_runner.py:91} INFO - Job 130: Subtask check_db
[2024-08-05T00:20:01.642+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=26338) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-05T00:20:01.643+0000] {standard_task_runner.py:64} INFO - Started process 26344 to run task
[2024-08-05T00:20:01.688+0000] {task_command.py:426} INFO - Running <TaskInstance: ingest_api_data.check_db scheduled__2024-08-05T00:10:00+00:00 [running]> on host 52830206e1a2
[2024-08-05T00:20:01.771+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='MS_sIX2Rm@somosyoung.com.br' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ingest_api_data' AIRFLOW_CTX_TASK_ID='check_db' AIRFLOW_CTX_EXECUTION_DATE='2024-08-05T00:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-05T00:10:00+00:00'
[2024-08-05T00:20:01.774+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-05T00:20:01.787+0000] {base.py:84} INFO - Using connection ID 'postgres_dw' for task execution.
[2024-08-05T00:20:01.801+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2024-08-05T00:20:01.806+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-05T00:20:01.807+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-05T00:20:01.816+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ingest_api_data, task_id=check_db, run_id=scheduled__2024-08-05T00:10:00+00:00, execution_date=20240805T001000, start_date=20240805T002001, end_date=20240805T002001
[2024-08-05T00:20:01.857+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-05T00:20:01.879+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-05T00:20:01.881+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
